crates/swc_ecma_minifier/tests/terser/compress/functions/issue_2620_4/output.js

```diff
@@ -1,4 +1,2 @@
 var c = "FAIL";
-var a, NaN;
-a = 0, c = "PASS";
```

Comment:
```js
var c = "FAIL";
(function () {
    function f(a, NaN) {
        function g() {
            switch (a) {
                case a:
                    break;
                case ((c = "PASS"), NaN):
                    break;
            }
        }
        g();
    }
    f(0 / 0);
})();
console.log(c);
```
This case failed for another reason.
It matched the first branch, but `NaN !== NaN`, so this is a bug.